Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -89,6 +89,25 @@ WI.ColorPicker = class ColorPicker extends WI.Object
 
         this._color = WI.Color.fromString("white");
 
+        if (colorPalette) {
+            let colorSwatchDisplay = this._element.appendChild(document.createElement("div"));
+            colorSwatchDisplay.classList.add("color-swatch-display");
+
+            let swatchesTitle = colorSwatchDisplay.appendChild(document.createElement("h2"));
+            swatchesTitle.innerText = WI.UIString("Variables", "Variables", "Title of swatches section in Color Picker");
+            
+            let swatchWrapper = colorSwatchDisplay.appendChild(document.createElement("ul"));
```

Comment:
ditto above about naming (though in this case it'd maybe be something like `variableColorSwatchesListElement`)